packaging/fedora/snapd.spec

```diff
@@ -901,8 +901,9 @@ make -C data -k check
 %license COPYING
 %dir %{_libexecdir}/snapd
 # For now, we can't use caps
-# FIXME: Switch to "%%attr(0755,root,root) %%caps(cap_sys_admin=pe)" asap!
-%attr(4755,root,root) %{_libexecdir}/snapd/snap-confine
+# TODO:nonsetuid: update caps
+# TODO:nonsetuid: use define
+%caps(cap_dac_override,cap_dac_read_search,cap_sys_admin,cap_sys_chroot,cap_chown,cap_fowner,cap_sys_ptrace,cap_setuid,cap_setgid=p) %{_libexecdir}/snapd/snap-confine
```

Comment:
I wonder if we can somehow reuse the file with list of caps. I doubt it but I wanted to double-check.

Reply:
need to experiment a little with this